src/components/pull-to-refresh.tsx

```diff
@@ -211,13 +211,15 @@ const PullToRefresh: React.FC<PullToRefreshProps> = ({
 
     // Container has not been dragged enough, put it back to it's initial state
     if (!pullToRefreshThresholdBreached) {
-      pullDownRef.current!.style.visibility = 'hidden';
+      if (pullDownRef.current) pullDownRef.current!.style.visibility = 'hidden';
```

Comment:
I believe we can remove the non-null assertion operator here:

`if (pullDownRef.current) pullDownRef.current.style.visibility = 'hidden';`